cdci_data_analysis/analysis/parameters.py

```diff
@@ -303,7 +303,7 @@ def set_par(self, value, units=None):
         if units is not None:
             self.units = units
         self.value = value
-        return self.get_value_in_default_format()
+        return self.value
```

Comment:
This is exactly how it used to before, and we changed such behavior [here](https://github.com/oda-hub/dispatcher-app/pull/280/commits/30e2399977e2b1130178de620f813c3b8a666534), in order to solve the #279 .